Sources/SwiftFormat/API/Configuration.swift

```diff
@@ -241,20 +241,44 @@ public struct Configuration: Codable, Equatable {
     case always
 
     var isNever: Bool {
-      switch self {
-      case .never:
-        return true
-      default:
-        return false
-      }
+      self == .never
+    }
+
+    var isAlways: Bool {
+      self == .always
+    }
+  }
+
+  /// A private enum created to maintain backward compatibility with swift-format version 601.0.0,
+  /// which had a `MultilineStringReflowBehavior` enum without a String raw type.
+  ///
+  /// In version 601.0.0, the `reflowMultilineStringLiterals` configuration was encoded as an object
+  /// with a single key (e.g., `{ "never": {} }`) rather than as a string (e.g., `"never"`). This
+  /// enum allows decoding from both formats:
+  /// - First, we attempt to decode as a String using `MultilineStringReflowBehavior`
+  /// - If that fails, we fall back to this legacy format
+  /// - If both attempts fail, an error will be thrown
+  ///
+  /// This approach preserves compatibility without requiring a configuration version bump.
+  private enum LegacyMultilineStringReflowBehavior: Codable {
+    case never
+    case onlyLinesOverLength
+    case always
+
+    var isNever: Bool {
+      self == .never
     }
 
     var isAlways: Bool {
+      self == .always
+    }
```

Comment:
I don't think we need these since nothing else in the formatter will access them.